.github/workflows/release-drafter.yml

```diff
@@ -21,7 +21,10 @@ jobs:
           cache: gradle
       - name: Build with Gradle
         run: |
-          ./gradlew publishPublishMavenPublicationToLocalRepoRepository && tar -C build -cvf artifacts.tar.gz repository
+          ./gradlew publishPublishMavenPublicationToLocalRepoRepository
+      - name: Build shadowJar
+        run: |
+          ./gradlew shadowJar && tar -C build -cvf artifacts.tar.gz repository libs/opensearch-sql-jdbc-shadow-*.jar
```

Comment:
You might want to list the shadow jar name. Tried `tar -C build -cvf artifacts.tar.gz repository libs/opensearch-sql-jdbc-shadow-*.jar` in local, it failed with 
```
zsh: no matches found: libs/opensearch-sql-jdbc-shadow-*.jar

Reply:
Thanks. Fixed with 43e1af0bdf37438e76383faf8a9365c24f7c0070. 

Didn't want to specify the version and * didn't work with tar so I implemented a workaround.